### PR TITLE
Make prefixes for DC terms and type consistent

### LINKED ISSUE #241: Use prefixes `dcterms:` and `dctype:` consistently in SWEET
```
@prefix dcterms: <http://purl.org/dc/terms/> .
@prefix dctype: <http://purl.org/dc/dcmitype/> .
```

in order to avoid potential confusion where `dct:` may be used for either.

### PR DESCRIPTION
**Pull request by dr-shorthair**
fixes #241

### CODE REVIEW

**Review by brandonnodnarb — APPROVED** [review]
Good catch/fix.

**Review by nicholascar — APPROVED** [review]

**Comment by smrgeoinfo** [comment]
I see that ESIP\sweet\src\phenAtmoFront.ttl  has two prefix declarations

**Comment by smrgeoinfo** [comment]
dctype: is not used in sweet\src\realmHydroBody.ttl but prefix is declared. could be removed. Optional.

File: src/realmHydroBody.ttl [hunk 1]
```diff
@@ -19,7 +19,7 @@
 @prefix sostc: <http://sweetontology.net/stateChemical/> .
 @prefix sorea: <http://sweetontology.net/realm/> .
 @prefix sorelsc: <http://sweetontology.net/relaSci/> .
-@prefix dct: <http://purl.org/dc/dcmitype/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
```
Comment (smrgeoinfo) [inline src/realmHydroBody.ttl:22]: not used, could remove

**Review by smrgeoinfo — APPROVED** [review]
a couple of minor comments, not show stoppers.

**Review by lewismc — APPROVED** [review]
+1 to @smrgeoinfo comments to remove unused import. 
+1 to merge

File: src/realmHydroBody.ttl (hunk 1 above)
Comment (brandonnodnarb) [reply src/realmHydroBody.ttl:22]: Yes, IIRC from the prefix re-mapping exercise (I believe it was #165), there are unused prefixes scattered throughout.  

Counterpoint: perhaps `dctype` and `dcterm` should be included in every `ttl` file even if not _currently_ used.  As we ramp up adding definitions from multiple sources they will be used for provenance metadata and potentially reification statements.

As you say, these aren't show stoppers; just my 2p.

**Comment by brandonnodnarb** [comment]
I'll merge and create separate issues for unused imports.

**Pull request merged**